homeassistant/components/fan/__init__.py

```diff
@@ -183,6 +182,11 @@ def current_direction(self) -> Optional[str]:
         """Return the current direction of the fan."""
         return None
 
+    @property
+    def capability_attributes(self):
+        """Return capabilitiy attributes."""
+        return {ATTR_SPEED_LIST: self.speed_list}
```

Comment:
Speed list none? Attribute is dropped in the other cases.

Reply:
The default of speed list is an empty list.